manifests/bind_event.pp

```diff
@@ -0,0 +1,34 @@
+# Define: qpid::bind_event
+#
+# This define binds the event queue with the correct messages
+#
+# === Parameters
+#
+# $ssl_cert::                   SSL cert to use for qpid-config commands
+#
+# $queue::                      Name of the event queue
+#
+# $hostname::                   Set to localhost for qpid-config operations
+#
+# $port::			Port that qpid is listening on
+define qpid::bind_event(
+  $ssl_cert,
+  $queue,
+  $hostname = 'localhost',
+  $port = 5671
+) {
+  if($ssl_cert) {
```

Comment:
Indentation is kind of weird here, the body of the define should have all the members at the same level of indentation, e.g. this if block should be at the same leve as exec